locales/en/plugin__kubevirt-plugin.json

```diff
@@ -543,12 +543,12 @@
   "Guest agent is required": "Guest agent is required",
   "Guest login credentials": "Guest login credentials",
   "Guest management": "Guest management",
-  "Guest system access log": "Guest system access log",
-  "Guest system access logs are disabled at cluster": "Guest system access logs are disabled at cluster",
-  "Guest system access logs are disabled at VirtualMachine": "Guest system access logs are disabled at VirtualMachine",
-  "Guest system access logs disabled at cluster": "Guest system access logs disabled at cluster",
-  "Guest system access logs not ready. a restart required": "Guest system access logs not ready. a restart required",
+  "Guest system log": "Guest system log",
   "Guest system log access": "Guest system log access",
+  "Guest system logs are disabled at cluster": "Guest system logs are disabled at cluster",
+  "Guest system logs are disabled at VirtualMachine": "Guest system logs are disabled at VirtualMachine",
+  "Guest system logs disabled at cluster": "Guest system logs disabled at cluster",
```

Comment:
Because someone has added those messages to the UI. And it was not me. I am not brave enough to remove something that was designed and implemented this way. Also because this is not the subject of this PR.